### PR TITLE
playlist-feature

### PR DESCRIPTION
**Pull request by lomesshh**

### CODE REVIEW

File: src/frontend/context/playlistcontext.jsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  }
+};
+
+const PlaylistProvider = ({ children }) => {
+  const { token } = useAuth();
+
+  const [playliststate, playlistdispatch] = useReducer(reducer, initialValue);
+
+  console.log("ALl", playliststate.playlist);
+
```
Comment (romabulani) [inline src/frontend/context/playlistcontext.jsx]: Remove console logs

File: src/frontend/pages/Playlist/PlaylistCard.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            content="Remove from playlist"
+            theme="light"
+            followCursor={true}
+            plugins={[followCursor]}
+          >
+            <i
+              class="fa-solid fa-trash-can"
+              onClick={() => addVideoToPlaylist(item, playlist)}
+            ></i>
+          </Tippy>
```
Comment (romabulani) [inline src/frontend/pages/Playlist/PlaylistCard.jsx:42]: Tippy is new for me.. learnt something new

**Review by romabulani — APPROVED** [review]
- Clean code
- Good to merge

File: src/frontend/context/playlistcontext.jsx (hunk 1 above)
Comment (lomesshh) [reply src/frontend/context/playlistcontext.jsx]: done

**Review by shrey27 — APPROVED** [review]
Code is well structured and covers all the use cases
PR is ready to merge

**Pull request merged**